### ISSUE TITLE
[Feature] scrollbar on overflow y

### ISSUE THREAD

**Issue by NickvdMeij** [ShoppinPal/ng-mobile-menu#10]
I'm missing the side menu scroll when there is an overflow. its a simple ``` overflow:auto ``` but its nice to have ;)

**Comment by pulkitsinghal** [comment]
@NickvdMeij _ I'm not sure if the original author behind this effort - @jeff-french is going to work on this anymore, so if you want to take over, let us know. If @jeff-french is cool with that too then you can take over.

**Comment by jeff-french** [comment]
Yeah, that' s cool with me. I'm not using this anymore. I've switched my current projects to use https://github.com/dpiccone/ng-pageslide.

**Comment by pulkitsinghal** [comment]
@NickvdMeij - there you go! You can either switch to the good stuff that jeff is using or carry on with a fork or take ownership of this one.
